docs/advisors/checks/performance-pg-archiver-failing.md

```diff
@@ -15,16 +15,16 @@ archive_command = 'copy "%p" "C:\\server\\archivedir\\%f"'  # Windows
 
 which will copy archivable WAL segments to the directory `/mnt/server/archivedir` in the above example.
 
-The archive command should return a zero (0) code so PostgreSQL will assume the WAL was successfully archived and will remove it or recycle it, in this manner the WAL directory size will be consistent with no big differences. In the case the command returns a no zero code PostgreSQL won’t remove or recycle the WAL file and will retry the archive until it success, in the meantime the number of WAL files in the WAL directory will increase and thus the occupied space, with the risk of run out of space.
+The archive command should return a zero (0) code so PostgreSQL will assume the WAL was successfully archived and will remove it or recycle it. In this manner the WAL directory size will be consistent with no big differences. In the case the command returns a non zero code, PostgreSQL won’t remove nor recycle the WAL file and will retry the archive until it succeeds. In the meantime the number of WAL files in the WAL directory will increase and thus will the occupied space, with the risk of running out of space.
 
 
 ## Resolution
 
 From the PostgreSQL log we should be able to get insight about the failure, we can even try the command manually to verify the result. 
 
-Something to keep in mind is the archive command will be executed under the ownership of the same user that the PostgreSQL server is running as, so the paths, tools, scripts, etc., that are called or acceded from the archive command should be accessible for the user running the service (usually postgres).
+Something to keep in mind is that the same user that runs the PostgreSQL server (usually postgres), will execute the archive command, so the paths, tools, scripts, etc., that are called or acceded from the archive command should be accessible for the user running the service.
```

Comment:
I'm not sure if "**acceded**" would be the best wording here. I couldn't easily understand what it intends to say in this context. We may simplify a bit? We should remember that many no-native speakers will use this documentation, and using simpler daily vocabulary is a must.